src/components/functionselect/functionselect.js

```diff
@@ -55,8 +102,8 @@ angular.module('vlui')
           // reset field def
           // HACK: we're temporarily storing the maxbins in the pill
           pill.bin = selectedFunc === BIN ? true : undefined;
-          pill.aggregate = getAggrs(type).indexOf(selectedFunc) !== -1 ? selectedFunc : undefined;
-          pill.timeUnit = getFns(type).indexOf(selectedFunc) !== -1 ? selectedFunc : undefined;
+          pill.aggregate = getAggregates(type).concat(['count']).indexOf(selectedFunc) !== -1 ? selectedFunc : undefined;
```

Comment:
This concat is very weird.  You only use aggregate.all  once, why don't just you add count to `aggregates.all`?   (Add comment explain why there too.)